[AGM-1813] Create 'Solving Problems' section on AdGuard for Safari (#416)
AdguardTeam/KnowledgeBase: docs/adguard-for-safari/solving-problems/ads-not-blocked.md

---
title: Filtering doesn't work and ads are not blocked
sidebar_position: 2
---

:::info

This article is about AdGuard for Safari, which safeguards only your Safari browser. To protect your entire device, [download the AdGuard app](https://agrd.io/download-kb-adblock)

:::

> Sometimes, filtering may not work and ads will still show despite AdGuard for Safari and proper filters being enabled. This usually happens when something goes wrong with filter updates.

Reviewer: Сначала может на тестовую страницу отправлять?
https://adguard.com/test.html

> when something goes wrong with filter updates

обычно это из-за багов Сафари. Поэтому приходится контент-блокеры дёргать туда-сюда.

Author: @Alex-302 юзеру идти на тестовую и что дальше делать?

Reviewer: Проверить статус фильтрации
![image](https://github.com/AdguardTeam/KnowledgeBase/assets/8361299/d7740a19-ef60-4f2a-9bce-1616e8b14bfe)

Или явно написать про случай, когда перестала блокироваться реклама на тех сайтах, где ранее блокировалась. Чтобы юзер не тыкал контент блокеры если на каком-то одном сайте увидел рекламу.

Author: проверить статус фильтрации - и если Running то идти выключать контент-блокеры?
а если статус Not running то что делать?

про случай с рекламой на одном сайте ниже в этой статье описано, что он может отрепортить

Author: @Alex-302

Reviewer: Тогда второй вариант - написать, что это для случая, когда фильтрация вообще не работает("нигде").

Возможно, по пустым контент-блокерам в настройках приложения видно, что что-то не так. Давно не встречал у себя этой проблемы, не помню как выглядело (и можно ли было определить).

### How to fix

 1. Open Safari.
 2. Click *Safari* in the left upper corner menu.
 3. Click *Settings…* → *Extensions* tab
 4. Uncheck checkboxes for all 6 AdGuard content blockers, this is how it should look like:
    ![Unchecked content blockers](https://cdn.adtidy.org/content/Kb/ad_blocker/safari/adg-safari-unchecked-cbs.png)

>  5. Check checkboxes for all 6 AdGuard content blockers again.

Reviewer: После 4 желательно перезапустить Сафари.

Author: поправил

---

> title: Filtering doesn't work and ads are not blocked

Reviewer 2: косметическая правка, но хочется единообразия какого-то
```suggestion
title: Filtering doesn’t work and ads aren’t blocked
```

Reviewer 2: ну или наоборот, все раздельно

sidebar_position: 2
---

:::info

This article is about AdGuard for Safari, which safeguards only your Safari browser. To protect your entire device, [download the AdGuard app](https://agrd.io/download-kb-adblock)

:::

> Sometimes, filtering may not work and ads will still show despite AdGuard for Safari and proper filters being enabled. This usually happens when something goes wrong with filter updates due to Safari bugs.

Reviewer 2: а еще может быть надо просто обновить фильтры... многие юзеры не дожидаются пока конвертация завершится и закрывают приложение

Reviewer 2: по тексту вот так предлагаю подправить (не очень нравится will и being enabled)
```suggestion
In some cases, filtering may not work and ads may still be displayed even though AdGuard for Safari and the proper filters are enabled. This usually happens when something goes wrong filter updates due to Safari bugs.
```

> ### How to fix

Reviewer 2: хочется it дописать, но по желанию

 1. Open Safari.
 2. Click *Safari* in the left upper corner menu.

>  3. Click *Settings…* → *Extensions* tab

Reviewer 2: ```suggestion
 3. Click *Settings…* → *Extensions*.
```

Reviewer 2: ну и без стрелочки. если глаголы использовать, то как-то так это должно выглядеть: `Click *Settings...* and switch to the *Extensions* tab.`

Reviewer 2: короче пофиг, вот два предложенных варианта верные, остальные нет

>  4. Uncheck checkboxes for all 6 AdGuard content blockers, this is how it should look like:

Reviewer 2: - я бы добавила их названия. картинка может не прогрузиться
- плюс не очень нравится что ты отходишь от формата инструкции (this is how it should look like)

```suggestion
 4. Clear the checkboxes for AdGuard’s 6 content blocking extensions: *Custom*, *General*, *Other*, *Privacy*, *Security*, and *Social*.
```

>  4. Clear the checkboxes for AdGuard’s 6 content blocking extensions: *Custom*, *General*, *Other*, *Privacy*, *Security*, and *Social*.

Reviewer 2: вот тут дефис у контент-блокинг

    ![Unchecked content blockers](https://cdn.adtidy.org/content/Kb/ad_blocker/safari/adg-safari-unchecked-cbs.png)
 5. Restart Safari.

>  6. Check checkboxes for all 6 AdGuard content blockers again.

Reviewer 2: ```suggestion
 6. Return to *Settings...* → *Extensions* and re-enable the content blocking extensions.
```

>  6. Return to *Settings...* → *Extensions* and re-enable the content blocking extensions.

Reviewer 2: ну и тут...

Reviewer 2: (да я сначала написала неправильно извините)

>  7. Open AdGuard for Safari settings by clicking its icon beside the Safari's search bar.

Reviewer 2: хз тоже бы путь уточнила
```suggestion
 7. Click the AdGuard for Safari icon next to the search bar and open its settings.
```

>  8. Go to *Filters* tab.

Reviewer 2: либо the Filters tab, либо Filters
```suggestion
 8. Go to *Filters*.
```

Reviewer: как по мне - сейчас вполне ясно, сам так пишу, если у юзеров проблемы. Filters - название вкладки, выделено жирным для удобства чтения. Вдруг юзер начитался, и зачем-то полезет в каталог с фильтрами?

Author: мне тож каж the не обязательно

Reviewer 2: обязательно, ну :)

 9. Click *Check for updates*.

>  10. Do not close AdGuard for Safari window until filter update is finished.

Reviewer 2: то же самое — либо the window, либо без него
+ the filter update is complete либо вот так:

```suggestion
 10. Do not close AdGuard for Safari until the filters are updates.
```

Reviewer: закрыть окно и закрыть приложение - разные действия. Скорее всего это кусок иосного совета - там если переключиться на 20-30 сек на другое приложение, система прибьёт Адгард (не полностью, но процесс обновления нарушается).

Reviewer: гугл про артикли уже забыл
 ![image](https://github.com/AdguardTeam/KnowledgeBase/assets/8361299/c8c75a8b-6486-43bc-a7cd-e20995fbcb0d)

> If that didn't help and ads are still not blocked, please contact our support at support@adguard.com.

Reviewer 2: ```suggestion
If that doesn’t help and ads are still not being blocked, please contact our support at support@adguard.com.
```

или: If the problem persists,...

> If you have issues only on a particular page or website, you can report it right from the browser: click AdGuard for Safari icon beside the Safari's search bar and then click *Report an issue*.

Reviewer 2: ```suggestion
If you only have problems with a specific page or website, you can report it directly from the browser: click the AdGuard for Safari icon next to the Safari search bar, and then click *Report an issue*.
```
